mock/docs/site-defaults.cfg

```diff
@@ -463,6 +463,16 @@
 
 ## Use this to force foreing architecture (requires qemu-user-static)
 # config_opts['forcearch'] = None
+## mapping from target_arch (or forcearch) to arch in /usr/bin/qemu-*-static
+# config_opts['qemu_user_static_mapping'] = {                                                                                                                                              
+#     'aarch64': 'arm',
+#     'i386': 'i386',
+#     'i686': 'i386',
+#     'ppc64': 'ppc64',
+#     'ppc64le': 'ppc64le',
```

Comment:
I can only see qemu-user-static-ppc on my system

Reply:
This is about file not about package:
```
$ rpm -qf /usr/bin/qemu-ppc64le-static
qemu-user-static-ppc-7.0.0-10.fc37.x86_64
```

Reply:
Errrrr, indeed.   Sorry for the noise